UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -44,6 +45,10 @@ public partial class UndertaleRoomEditor : DataUserControl
 
         public static readonly PropertyInfo visualOffProp = typeof(Canvas).GetProperty("VisualOffset", BindingFlags.NonPublic | BindingFlags.Instance);
         private static readonly MainWindow mainWindow = Application.Current.MainWindow as MainWindow;
+        private static readonly Regex endNumberEx = new(@"\d+$", RegexOptions.Compiled);
+
+        // used for the flashing animation
```

Comment:
adding "when an element is selected" makes it a bit clearer on what kind of flashing anim it is.